src/detection/terminalfont/terminalfont.c

```diff
@@ -320,6 +320,28 @@ FF_MAYBE_UNUSED static bool detectKitty(const FFinstance* instance, FFTerminalFo
     return true;
 }
 
+static bool detectTerminator(const FFinstance* instance, FFTerminalFontResult* result)
+{
+    FFstrbuf fontName;
+    ffStrbufInit(&fontName);
+
+    FFpropquery fontQuery[] = {
+        {"font =", &fontName},
+    };
+
+    if(!ffParsePropFileConfigValues(instance, "terminator/config", 1, fontQuery))
+        return false;
+
+    if(fontName.length == 0)
+        ffStrbufSetS(&fontName, "Mono 8");
```

Comment:
The fontName can't be 0 length, because then ffParsePropFileConfig wouldn't return true

Reply:
Hmm it never returns false but i think this is cause of other config values i have like `title_use_system_font = False` or `title_font = MesloLGS Nerd Font Mono 9`

Full test config
```
[global_config]
  title_hide_sizetext = True
  title_transmit_bg_color = "#536a13"
  title_inactive_bg_color = "#5e5c64"
  enabled_plugins = LaunchpadBugURLHandler, LaunchpadCodeURLHandler, APTURLHandler, TerminalShot
  title_use_system_font = False
  title_font = MesloLGS Nerd Font Mono 9
[profiles]
  [[default]]
    audible_bell = True
    urgent_bell = True
    cursor_blink = False
    cursor_shape = ibeam
    cursor_color = "#aaaaaa"
    font = MesloLGSDZ Nerd Font Mono 10
    scrollback_infinite = True
    use_system_font = False
```

But i don't understand why fontName.length is still zero, except False is false and not "False" :smile: 

i'm not 100% sure if there is always a space before `font =` but i will try if config spaces are needed by terminator. if so i would query ` font =`

Is there a way to make the query a bit more complex like space is not needed before equal sign :thinking: in regex i would do something like /^\s*font\s?=/g :smile: 

But first i look at the terminator docs if config spaces are needed :)

Reply:
> The fontName can't be 0 length, because then ffParsePropFileConfig wouldn't return true

I think it only returns false if the config file is not found, if i read the code correctly :wink: if config is there but property not found fontName.length is 0

Reply:
i'm not 100% sure if there is always a space before font = but i will try if config spaces are needed by terminator. if so i would query " font ="

But, is there a way to make the query a bit more complex like space is not needed before equal sign.. thinking in regex i would do something like /^\s*font\s*=/g :smile:

Reply:
ffParsePropQuery is actually quite smart, with the key `font =` it will detect things like `  font=jeue` too.